hack/lib/serverless.bash

```diff
@@ -4,7 +4,8 @@ function ensure_serverless_installed {
   logger.info 'Check if Serverless is installed'
   local prev=${1:-false}
   if oc get knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}" >/dev/null 2>&1 && \
-     oc get knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1
+     oc get knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1 && \
+     oc get knativekafka.operator.serverless.openshift.io knative-kafka -n "${EVENTING_NAMESPACE}" >/dev/null 2>&1
```

Comment:
I just realized that this condition should only be considered if $INSTALL_KAFKA is true

Reply:
But, on the other hand, the install is idempotent, so, should be fine doing it twice in case serving and eventing is already installed and kafka isn't.

Again, I would change this as a follow up.

Reply:
PR against this PR's branch: https://github.com/matzew/serverless-operator/pull/3